defaults/helpers.go

```diff
@@ -18,3 +33,188 @@ func ociProfileConversionCheck(profile secprofile.Profile, entitlementID string)
 
 	return ociProfile, nil
 }
+
+// FIXME factor for all defaults package and even project-wide
+func testSpec() *specs.Spec {
```

Comment:
if this is only used for testing, we should move it to the test file - or is this intended to be a generic spec to be reused for defaults?

Reply:
this is for test purposes only, I'll move this in a test file